tests/test_linear_incident_uptake_model.py

```diff
@@ -89,7 +92,12 @@ def test_fit(frame):
     """
 
     data = iup.IncidentUptakeData(frame)
-    model = iup.models.LinearIncidentUptakeModel().fit(data, ("geography",))
+    model = iup.models.LinearIncidentUptakeModel().fit(
+        data,
+        [
+            "geography",
+        ],
```

Comment:
```suggestion
        ]
```
if you cut off this comma, then ruff will probably collapse this to a single line